internal/version/version.go

```diff
@@ -1,19 +1,23 @@
 package version
 
-import "runtime/debug"
-
-const ProtocolVersion = "v10.0.0"
+import (
+	"fmt"
+	"runtime/debug"
+)
 
 var (
-	// AppVersion is set by go build -ldflags
-	AppVersion = "Unspecified"
-
-	// AppGitCommit is set by go build -ldflags
+	// set by go build -ldflags
```

Comment:
I did that comment per variable, because that in interpreted as variable description by IDEs
maybe you can try to make var() block and comment whole block

Reply:
Ah ok, sounds logical. I've added the comments to the vars.